### PR TITLE
check no-drag better

### PR DESCRIPTION
**Pull request by sanyueyu**
i think this is better

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -39,9 +39,10 @@ class DragSortableList extends React.Component {
 
   componentDidMount() {
     const draggableChildrenSelector = '#' + this.ref + '> .draggable'
+    const noDragEle = event.target.querySelector('.no-drag');
     const ignoreNoDrag = fun => event => {
       const mouseElement = document.elementFromPoint(event.clientX, event.clientY)
-      if(mouseElement && !mouseElement.classList.contains('no-drag') && mouseElement.parentNode && !mouseElement.parentNode.classList.contains('no-drag')) {
+      if(!noDragEle || (noDragEle&&!noDragEle.contains(mouseElement))) {
```
Comment (tom-s) [inline src/index.js]: I think
````
if(!noDragEle || !noDragEle.contains(mouseElement)) {
````
would be enough ?

Comment (tom-s) [reply src/index.js]: we were checking the parent for the class as well before, you don't seem to do it anymore. Does that mean it was unnecesary ? (it's very possible ^^)

Comment (sanyueyu) [reply src/index.js]: no , I suggest you think about mine idea，It better check the drag experience

Comment (sanyueyu) [reply src/index.js]: "if(!noDragEle || !noDragEle.contains(mouseElement)) {"
 is better

Comment (sanyueyu) [reply src/index.js]: "if(!noDragEle || !noDragEle.contains(mouseElement)) {"
is better

**Pull request closed without merging**